modules/class-simplechart-save.php

```diff
@@ -93,7 +93,7 @@ protected function _do_save_post( $post ) {
 		foreach ( array( 'chartData', 'chartOptions', 'chartMetadata', 'googleSheetId' ) as $field ) {
 			if ( ! empty( $_POST[ 'save-' . $field ] ) ) {
 				// sanitize field name w/ esc_attr() instead of sanitize_key() because we want to preserve uppercase letters
-				update_post_meta( $post->ID, 'save-' . esc_attr( $field ), sanitize_text_field( wp_unslash( $_POST[ 'save-' . $field ] ) ) );
+				update_post_meta( $post->ID, 'save-' . esc_attr( $field ), sanitize_text_field( $_POST[ 'save-' . $field ] ) );
```

Comment:
You're going to need to tell WPCS to ignore the `wp_unslash` rule here. There is an open issue about false positives for this sniff with functions that expect slashed data, of which `update_post_meta` is one:

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/172